### PR TITLE
ci: added CLA Assistant workflow

### PR DESCRIPTION
**Pull request by janniclas**
Closes https://github.com/fraunhofer-iem/spha/issues/43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,46 @@
...
-The [gradle plugin](https://github.com/cortinico/ktfmt-gradle#how-to-use-) is installed for all submodules through the
-conventions plugin. It adds two gradle tasks
-`./gradlew ktfmtCheck` and `./gradlew ktfmtFormat`. The gradle plugin is configured to use `kotlinlang`.
-ktfmt has an [IntelliJ plugin](https://plugins.jetbrains.com/plugin/14912-ktfmt) which automatically replaces IntelliJ's
-format command and applies ktfmt's formatting. The IntelliJ plugin must be configured to use `kotlinlang` under the
-plugin's settings. 
\ No newline at end of file
+## SPHA Library
+
+This project contains SPHA's core library. Its main purpose is to calculate a products health score
```
Comment (AnakinRaW) [inline README.md:12]: "measure" instead of calculate?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,46 @@
...
+based on a given `KpiHierarchy` and `RawValueKPIs`. Further, it provides the possibility to transform
```
Comment (AnakinRaW) [inline README.md:13]: Instead of "it" "SPHA" or "The Library"

File: README.md [hunk 3]
```diff
@@ -1,13 +1,46 @@
...
+tool results into our internal `RawValueKPI` format. The transformation is handled by dedicated tool
+adapters.  
+With the [SPHA CLI Tool](https://www.github.com/fraunhofer-iem/spha-cli) we have created an executable tool using this
```
Comment (AnakinRaW) [inline README.md:16]: Instead of "we have created" something like "there is a CLI tool" or "you can use the CLI tool"

File: CLA.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# Fraunhofer IEM Contributor License Agreement
+
+Thank you for your interest in contributing to the open-source software project Software Product Health Assistant (hereinafter referred to as “the Project”) maintained by Fraunhofer Institute for Applied and Integrated Security (hereafter “us” or “IEM”).
```
Comment (AnakinRaW) [inline CLA.md:3]: Institute name is wrong ‼️

File: CLA.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+# Fraunhofer IEM Contributor License Agreement
+
+Thank you for your interest in contributing to the open-source software project Software Product Health Assistant (hereinafter referred to as “the Project”) maintained by Fraunhofer Institute for Applied and Integrated Security (hereafter “us” or “IEM”). 
+
+The purpose of this Contributor Agreement ("Agreement") is to clarify the intellectual property license granted by persons or entities who provide code to IEM. When the Contribution is made on behalf of an entity, the term "you" in this Agreement means both the person accepting this Agreement and the entity that employs that person.
+This license is for your protection as a Contributor as well as the protection of the Fraunhofer Institute IEM and its users; it does not change your rights to use your own Contributions for any other purpose.
```
Comment (AnakinRaW) [inline CLA.md:6]: "Fraunhofer Institute IEM" is that correct? Isn't that the same as "ISBN Number"?

**Review by AnakinRaW — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (matzebecker) [reply README.md:13]: changed to "SPHA"

File: CLA.md (hunk 4 above)
Comment (matzebecker) [reply CLA.md]: fixed institute name

File: README.md (hunk 1 above)
Comment (matzebecker) [reply README.md:12]: I think calculate is correct as well, hence I leave it as it is

File: README.md (hunk 2 above)
Comment (matzebecker) [reply README.md]: changed to "SPHA"

**Review by AnakinRaW — APPROVED** [review]
thanks

**Pull request merged**